CONTRIBUTING.md

```diff
@@ -42,4 +42,4 @@ rockcraft pack -v
 version=$(yq '.version' rockcraft.yaml)
 sudo skopeo --insecure-policy copy oci-archive:gocert_${version}_amd64.rock docker-daemon:gocert:${version}
```

Comment:
Can you please update this also to use the `skopeo` bundled with `rockcraft`?

Reply:
done, thanks for the catch